app/lib/report_helper.rb

```diff
@@ -193,7 +193,7 @@ def all_categories
   end
 
   def all_genus
-    genus_taxids = TaxonLineage.distinct.pluck(:genus_taxid).join(', ')
+    genus_taxids = TaxonLineage.distinct.limit(20).pluck(:genus_taxid).join(', ')
```

Comment:
this seems a bit arbitrary. the right way to do it should be a remote typeahead. just getting the arbitrary 20 genus ids doesn't seem to be right.